components/action-mgt/org.wso2.carbon.identity.action.management/pom.xml

```diff
@@ -178,14 +183,14 @@
                                             <!-- Temporarily decreasing this.
                                             Will be improved upon in a follow-up effort.
                                             Related issue: https://github.com/wso2/product-is/issues/21368 -->
-                                            <minimum>0.70</minimum>
+                                            <minimum>0.77</minimum>
```

Comment:
Target here should be 0.8

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6119/commits/931588f41d06bd38d14dd21ecff6b83e13fcc841